README.md

```diff
@@ -184,6 +184,19 @@ CREATE TABLE http (
 )
 ```
 
+#### Custom request/response processing
```

Comment:
I'm not sure if this is accurate title.
Title suggest at least to me that we can customize processing of request and response, for example process request before sending it to end point.

The nature for this callback feature is to perform a side output action, not related with main processing "pipeline".
So its not an actual request/response processing.

Reply:
fixed